app/views/coronavirus_landing_page/components/hub/_guidance_section.html.erb

```diff
@@ -8,7 +8,7 @@
           font_size: 19,
           margin_bottom: 5
         } %>
-        <ul class="covid__list">
+        <ul class="covid__list govuk-!-margin-bottom-8">
```

Comment:
Is it only this `covid__list` that requires this margin bottom or could it applied to the default for this class? We try not to use the spacing override classes unless it's actually needed to override something.

Reply:
It is only this `covid__list`, yeah. The default values for `covid__list` margins are different.
An alternative would be to use a modifier class instead of the spacing override class. 
But the idea of adding new CSS to basically do the same  thing as the existing spacing override class doesn't make much sense to me.

Reply:
Yeah, this is the party line on this but I actually agree. Don't worry.